src/components/AdditionColumn/AdditionColumn.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { act, fireEvent, render, screen } from "@testing-library/react";
+import { TestRendererWithContext } from "../../utils/testRendererWithContext";
+
+import AdditionColumn from "./AdditionColumn";
+
+import "jest-styled-components";
+
+beforeEach(() => {
+  act(() => {
+    render(<AdditionColumn />, {
+      wrapper: TestRendererWithContext,
+    });
+  });
+});
+
+describe("<AdditionColumn />", () => {
+  it("Default render - It should render closed state of the component", () => {
```

Comment:
It is a very small thing and you don't need to change. 
I think it is better to write it("should render closed state of the component", (() => {}) when you use it.

Reply:
I don't quite understand what you mean, could you elaborate on that a bit further? (Don't text me in LINE with Japanese BTW😂)

Reply:
if you write test with test function.
`test("passed projectId - it should be rendered correctly, () => {})` 
> test passed projectId - it should be rendered correctly.

if you write test with it function
`it("Default render - It should render closed state of the component", () => {})` 
> it Default render - It should closed state of the component. 

As english sentence it is not grammarly correct. so It should be something like
`it("Default render - It should render closed state of the component", () => {})` 
> it should closed state of the component.

Reply:
Ah makes sense! Yeah I mixed up with `test` and `it` honestly - I'll change `it` to `test` in this file. Thanks for the advice👍